specification/monitor/resource-manager/microsoft.insights/2017-04-01/actionGroups_API.json

```diff
@@ -459,6 +473,55 @@
                     "description": "The URI where webhooks should be sent."
                 }
             },
+            "ItsmReceiver": {
```

Comment:
I believe this is the root cause of [the failures from our CI system](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/310273380#L725). I think there's a curly-brace missing above this line.